charts/aws-ssm-agent-installer/values.yaml

```diff
@@ -10,3 +10,5 @@ tolerations:
     operator: Exists
   - effect: NoSchedule
     operator: Exists
+imagePullSecrets:
```

Comment:
imagePullSecrets: [] else it will give error

Reply:
Yes, done.